addons/project/data/mail_template_data.xml

```diff
@@ -103,7 +103,7 @@
 
         <!-- You have been assigned email -->
         <template id="project_message_user_assigned">
-    <span>Dear <t t-esc="assignee_name"/>,</span>
+    <span><br/>Dear <t t-esc="assignee_name"/>,</span>
```

Comment:
```suggestion
    <br/>
    <span>Dear <t t-esc="assignee_name"/>,</span>
```
You could also do something like this, to me it seems clearer :thinking:

Reply:
Changes are done, 
Thank you for your suggestion. :slightly_smiling_face: